lib/themes/dosomething/paraneue_dosomething/js/components/SlideshowController.js

```diff
@@ -1,14 +1,26 @@
+import { sendEvent } from '../utilities/Analytics';
+
 const React = require('react');
 
 /**
  * SlideshowController Component
  * <SlideshowController />
  */
 class SlideshowController extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.finish = this.finish.bind(this);
+  }
+
+  finish() {
```

Comment:
@DFurnes maybe we should start a further "React" subsection under the Javascript section on the code-style repo?